pkg/lintcontext/context.go

```diff
@@ -1,22 +1,66 @@
 package lintcontext
 
 import (
+	"encoding/json"
+
 	"golang.stackrox.io/kube-linter/internal/k8sutil"
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
 // ObjectMetadata is metadata about an object.
 type ObjectMetadata struct {
 	FilePath string
-	Raw      []byte
+	Raw      []byte `json:"-"`
 }
 
 // An Object references an object that is loaded from a YAML file.
 type Object struct {
 	Metadata  ObjectMetadata
-	K8sObject k8sutil.Object
+	K8sObject k8sutil.Object `json:"-"`
+}
+
+// K8sObjectName contains just enough information about k8s object to allow identifying and locating it.
```

Comment:
"Just enough information" is true right now, but it doesn't necessarily have to be true. We may in the future add more information to this struct if it is convenient. I would just say that it is a conveniently JSON-serializable representation of the object.

Further, I think we should also print out labels and annotations, since that's also standard object metadata that's on every object.

Reply:
Sure. Rephrased, please check